front_end/helper.py

```diff
@@ -110,15 +110,12 @@ def exec_code(settings_dict, code, exercise_basics, exercise_details, request=No
     response = requests.post(f"http://127.0.0.1:{middle_layer_port}/exec/", json.dumps(data_dict), timeout=timeout)
 
     response_dict = json.loads(response.content)
-    return response_dict["text_output"], response_dict["image_output"]
-
-def check_exercise_output(exercise_details, actual_text, actual_image):
-    if exercise_details["back_end"] == "any_response" and (len(actual_text) > 0 or len(actual_image) > 0):
-        return "", True
-
-    if exercise_details["output_type"] == "txt":
-        expected_text = exercise_details["expected_text_output"]
+    # 'text_output' and 'image_output' refer to the output produced by the code written by the student, while 'tests' is a dict containing image and text outputs specific to each test case written by the instructor
+    # tests must be converted from json to a list of dicts
```

Comment:
This is super nitpicky, but please use correct capitalization and punctuation in comments.